helpers/components/addInSplashAds.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const _ = require('lodash');
```

Comment:
Better to only pull in what you need here, rather than the whole lodash library. e.g.:
```
const _forEach = require('lodash/forEach');

...

_forEach(content, function (component) {
```